shim/src/realm.js

```diff
@@ -1,61 +1,226 @@
-import { createSandbox } from "./sandbox.js";
-import { sanitize } from "./sanitize.js";
+import { createSandbox, setSandboxGlobalObject } from "./sandbox.js";
 import { evaluate } from "./evaluate.js";
-import { getEvaluators } from "./evaluators.js";
 import { getStdLib } from "./stdlib.js";
 import { getIntrinsics } from "./intrinsics.js";
-import { proxyHandler } from "./proxy.js";
+import { assert, IsCallable } from "./utils";
 
-const RealmToSandbox = new WeakMap();
+const RealmRecord = Symbol('Realm Slot');
```

Comment:
this it to match the way the spec defines those records so we can keep them in sync with the spec easily.